src/main/webapp/app/shared/metis/post/post.component.html

```diff
@@ -1,17 +1,30 @@
-<jhi-post-header
-    [previewMode]="previewMode"
-    [readOnlyMode]="readOnlyMode"
-    [posting]="posting"
-    [isCourseMessagesPage]="isCourseMessagesPage"
-    [isCommunicationPage]="isCommunicationPage"
-    [hasChannelModerationRights]="hasChannelModerationRights"
-    (isModalOpen)="displayInlineInput = true"
-    [lastReadDate]="lastReadDate"
-/>
+<div class="post-header-container">
```

Comment:
You don't necessarily need to create a extra class. You can use the bootstrap classes `d-flex` and `align-items-center` instead